### PR TITLE
Pull request #2

### PR DESCRIPTION
**Pull request by pel7373**

### CODE REVIEW

**Comment by Volodymyr95** [comment]
.idea in pr

File: src/main/java/ufo/AcceptChallengeService.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import org.slf4j.LoggerFactory;
+import ufo.answers.Answer;
+
+import static ufo.Constants.*;
+
+public class AcceptChallengeService {
+    private static final Logger LOGGER = LoggerFactory.getLogger(
+            AcceptChallengeService.class);
+
+    public Answer call(boolean isAccepted) {
```
Comment (Volodymyr95) [inline src/main/java/ufo/AcceptChallengeService.java:13]: why call?

File: src/main/java/ufo/AcceptChallengeService.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        Answer answer;
+
+        sb.append("Answer was received: ")
+                .append(isAccepted)
+                .append("; message: ");
+
+        if (isAccepted) {
+            answer = new Answer(ACCEPT_CHALLENGE_ACCEPTED, ACCEPT_CHALLENGE_NEXT_PAGE);
+            sb.append(ACCEPT_CHALLENGE_ACCEPTED)
+                    .append("; next page: ")
```
Comment (Volodymyr95) [inline src/main/java/ufo/AcceptChallengeService.java:24]: why u added this parameter?

File: src/main/java/ufo/web/AcceptChallengeServlet.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+import java.util.Optional;
+
+import static ufo.Constants.*;
+
+@WebServlet("/acceptChallengeServlet")
```
Comment (Volodymyr95) [inline src/main/java/ufo/web/AcceptChallengeServlet.java:20]: don't use word servlet in url's, and rename to accept-challenge, cammel case is not used in url

**Comment by pel7373** [comment]
> .idea in pr

This is no longer the case. It used to be, now it doesn't

File: src/main/java/ufo/AcceptChallengeService.java (hunk 2 above)
Comment (pel7373) [reply src/main/java/ufo/AcceptChallengeService.java:24]: What parameter? ACCEPT_CHALLENGE_ACCEPTED in the logger? If so, we can find out the state of the system at this  step.

File: src/main/java/ufo/AcceptChallengeService.java (hunk 1 above)
Comment (pel7373) [reply src/main/java/ufo/AcceptChallengeService.java:13]: https://www.youtube.com/watch?v=M0XteHOwMZc , 43:13 and later - u named it so. Rename?

**Pull request closed without merging**